docs/source/install-tutorial.rst

```diff
@@ -115,3 +138,43 @@ Supervisord config i.e. in ``/etc/supervisor/conf.d/bepasty.conf``:
   redirect_stderr = true                                        ; Save stderr in the same log
 
 Finally reload supervisor: `service supervisor reload`
+
+In your browser go to your server address: ``http://bepasty.example.org``
+
+You should now have a running bepasty server on port 80 (HTTP) of your server.
+
+Places to look when troubleshooting:
+
+::
+
+  journalctl -u nginx.service
+  journalctl -u supervisor.service
+  less +F /home/bepasty/logs/nginx-access.log
+  less +F /home/bepasty/logs/nginx-error.log
+  less +F /home/bepasty/logs/gunicorn_supervisor.log
+
+Important notes:
+
+* If you copied the file from the ``bepasty/config.py`` it will have
+  a "class Config" in it and all the settings are inside that class. This is
+  **not** what you need. Due to how flask config files work, you need to
+  remove the class statement and outdent all the settings, so you just have
+  global KEY = VALUE statements left on the top level of the config file.
+  If you wish to use a script to do this in one go - please see the Tips & Tricks below.
+* When adding additional users in ``bepasty/config.py`` make sure you have
+  ``'user_secret': 'correct,list,of,permissions',``
+  any missing ``'' : '',`` will result in the bepasty server not starting.
+  
+Tips & Tricks:
+* To convert the ``src/bepasty/config.py`` to ``/home/bepasty/bepasty.conf`` in one easy go.
+  On the command line as the bepasty user:
+  # create trim alias
+  ``alias trim="awk '{\$1=\$1};1'"``
+  # remove the top 11 lines + trim spaces at beginning of lines
```

Comment:
this looks like something that is going to break if we ever change the header of that file.

Reply:
so, just remove that, please.

Reply:
Fair enough.  Do you have any suggestions on an alternative approach to automating this given the amount of manual effort needed to get the config file into a suitable format and the consequences and errors if not?

Reply:
the install tutorial is for a manual installation, so it does not need to do everything automated.